### PR TITLE
Mount $HOME:$HOME by default in podman machine init

### PR DESCRIPTION
**Pull request by baude**
By default, mount the user's homedir into the vm at the same location.
any volumes defined on the cli will then override the default mount.
Also, added --no-volumes to disable volume mounts alltogether.

Signed-off-by: Brent Baude <bbaude@redhat.com>

[NO NEW TESTS NEEDED]

### CODE REVIEW

**Comment by baude** [comment]
do not merge this .... @rhatdan is going to make a field for container's conf and we will read the default volume mounts from there.  I will rework this PR then.

**Comment by rhatdan** [comment]
LGTM
@containers/podman-maintainers PTAL
@giuseppe PTAL

**Review by giuseppe — APPROVED** [review]
/lgtm

**Pull request merged**
